### PR TITLE
Change name through onfleet api

### PR DESCRIPTION
**Pull request by michellejkwon**
### What does this PR do?
Updates to profile (currently only company name), also appears on Onfleet. In marketplace_users_controllers, makes call to OnfleetAPI before updating in local database.
Can no longer update phone number (this is their unique id on onfleet)

### Status
READY

### Related PRs
see PR on replate-marketplace: https://github.com/calblueprint/replate-marketplace/pull/22

### Description of Testing Done
Manual testing (saw change in Onfleet)

### Screenshots
screen shot 2018-02-12 at 8 03 38 pm
screen shot 2018-02-12 at 8 06 01 pm

CC: @michaeldo2

### CODE REVIEW

File: app/controllers/api/marketplace/marketplace_users_controller.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ def show
   end
 
   def update
+    onfleet_worker_result = OnfleetApi.update_worker(params[:onfleet_driver_id], onfleet_user_params)
```
Comment (michaeldo2) [inline app/controllers/api/marketplace/marketplace_users_controller.rb]: Lets use the result `onfleet_worker_result` to determine if we should even attempt to update the user on our db. If for some reason `OnFleetApi`'s request fails, we should right away tell the user something went wrong so we can try to keep our database as synced as possible with OnFleet's.

File: db/schema.rb [hunk 2]
```diff
@@ -238,8 +238,8 @@
     t.datetime "updated_at",                     null: false
     t.text     "description"
     t.integer  "trays_donated"
-    t.boolean  "paid",           default: false, null: false
     t.string   "short_id",       default: "",    null: false
+    t.boolean  "paid",           default: false, null: false
```
Comment (michaeldo2) [inline db/schema.rb]: What is this field for? I can't seem to find any changes involving it

Comment (nhiquach) [reply db/schema.rb]: Migrations do something weird with git... probably something from replate developers, not us

Comment (caseytakahashi) [reply db/schema.rb]: Yeah, I also found this odd, but we didn't do anything to change this field, so wouldn't know how to undo the change

Comment (nhiquach) [reply db/schema.rb]: since schema.rb is autogenerated this happens, nothing to worry about/no need to undo

Comment (michaeldo2) [reply db/schema.rb]: ah i see no worries! That makes sense

**Pull request merged**